tests/js/jest-pegjs-transform.js

```diff
@@ -1,10 +1,7 @@
-/* eslint-env node */
-
-const crypto = require('node:crypto');
-const peggy = require('peggy');
+'use strict';
 
 function getCacheKey(fileData, _filePath, config, _options) {
-  return crypto
+  return require('node:crypto')
```

Comment:
There's a small performance penalty when inlining require calls. Should be able to just keep imports at the top?

Reply:
I moved it because the var name conflicted with the node global 'crypto' (according to the ts dfns file). 

So nbd to move the require back up. I'll give it a different var name from what it had before.